src/model/mod.rs

```diff
@@ -11,6 +11,7 @@ pub mod puzzles;
 pub mod simuls;
 pub mod studies;
 pub mod tablebase;
+pub mod token;
```

Comment:
Also not documented by me - I try to keep the naming as close to the api docs as I can. OAuth is the name of the category in this case.

```suggestion
pub mod oauth;
```